addon-test-support/index.js

```diff
@@ -11,4 +12,7 @@ import { createSandbox, restoreSandbox } from './sinon-sandbox';
 export default function setupSinon(testEnvironment = self.QUnit) {
   testEnvironment.testStart(createSandbox);
   testEnvironment.testDone(restoreSandbox);
+
+  sinon.assert.pass = (assertion) => self.QUnit.assert.ok(true, assertion);
```

Comment:
Is it ok that we are importing `sinon` here? Should this be something passed in? Should it be `self.sinon`?

Reply:
I think it's fine to import sinon here, because it's a required dependency through ember-sinon. Even beyond that, though, we shouldn't have to expect people to use this package if they're not using sinon. That's not a workflow we need to support, so import is fine here!

Reply:
Should we import QUnit here or is it required to be grabbed from `self` for some reason?

Reply:
I just grabbed it rom `self` because you did on line 12. Which also means we could probably swap it to `testEnvironment` instead. I'm without opinion, which would you prefer?

Reply:
Ah, great point. I think we should definitely swap it to testEnvironment to make sure that still works, and then maybe separate from this change we should consider whether we can just do

`import * as QUnit from 'qunit';`

I don't trust my past self to have made that decision wisely, but maybe there was a reason for it? XD

Reply:
Opened an issue to separate that question from this PR: https://github.com/elwayman02/ember-sinon-qunit/issues/530